### PR TITLE
implemented true/false functions for isSubmitted

### PR DESCRIPTION
**Pull request by jgarcia19**

### CODE REVIEW

**Review by samuel-ping — DISMISSED** [review]
Line 73: This seems a little out of place, especially since we don't declare any other functions outright with "let." For the sake of consistency, can you remove this let? But other than that, everything looks a-ok to me! (I also want you to push new changes to test the pipeline :p)

**Comment by jgarcia19** [comment]
@samuel-ping the only reason i put that let there is because VS code was showing an error at line 73 without it. If it runs fine without that let feel free to remove it!

File: server/databaseFunctions.test.js [hunk 1]
```diff
@@ -49,9 +50,27 @@ describe("testing basic endpoints", () => {
...
+    expect(res.statusCode).toEqual(200);
+    expect(res.body.isSubmitted).toEqual(false);
+    done();
+  });
+  
+  
 });
 
-removeAllCollections = async () => {
+let removeAllCollections = async () => {
```
Comment (samuel-ping) [inline server/databaseFunctions.test.js]: and delete this let

File: server/databaseFunctions.test.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const request = supertest(app);
 const testTeamID = mongoose.Types.ObjectId();
 var testSubmissionId; // this id will be automatically generated for the below submission after that submission is added to the database
 const testSubmission = {
+  projectID: testTeamID,
```
Comment (samuel-ping) [inline server/databaseFunctions.test.js]: delete this

File: server/app.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const express = require("express");
+const { update } = require("./models/submissionSchema.model");
```
Comment (samuel-ping) [inline server/app.js:2]: whats this?

File: server/databaseFunctions.test.js [hunk 4]
```diff
@@ -49,6 +49,24 @@ describe("testing basic endpoints", () => {
     expect(res.statusCode).toEqual(200);
     done();
   });
+  
+
+  it("sets isSubmitted of given submission to true", async (done) => {
+    const res = await request.patch(`/api/submissions/${testTeamID}/${testSubmissionId}/setTrue`);
```
Comment (samuel-ping) [inline server/databaseFunctions.test.js]: I think it would be clearer if you called the route /submit and /unsubmit

**Review by samuel-ping — CHANGES_REQUESTED** [review]
lol sorry for so many comments, take a look at my small comments

**Pull request merged**